packages/cysync-core/src/store/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { connect, ConnectedProps } from 'react-redux';
+import { AppDispatch, RootState } from './store';
+
+const mapStateToProps = (state: RootState) => ({
+  lang: state.lang,
+});
+
+const mapDispatchToProps = (dispatch: AppDispatch) => ({
+  dispatch,
+});
+
+export const defaultConnector = connect(mapStateToProps, mapDispatchToProps);
```

Comment:
Redux recommends using hooks instead of connect api
https://react-redux.js.org/api/connect